docker/dockerfile_generator.sh

```diff
@@ -0,0 +1,309 @@
+#!/bin/bash
+
+OPENJ9=false
+BUILD=false
+COMMENTS=false
+DFDIR=
```

Comment:
Whats DF?...

Reply:
Dockerfile. But I'll make it clearer :-)